v3.0/how-to/upgrade/from-previous-version.md

```diff
@@ -6,29 +6,23 @@ aliases: ['/docs-cn/op-guide/tidb-v3.0-upgrade-guide/','/docs-cn/v3.0/how-to/upg
 
 # TiDB 3.0 升级操作指南
 
-本文档适用于从 TiDB 2.0 版本（v2.0.1 及之后版本）或 TiDB 2.1 版本（v2.1 RC1 及之后版本）升级到 TiDB 3.0 或 master 版本。目前，TiDB 3.0 和 master 版本兼容 [TiDB Binlog Cluster 版本](/v3.0/reference/tidb-binlog/overview.md)。
+本文档适用于从 TiDB 2.0 版本（v2.0.1 及之后版本）或 TiDB 2.1 版本（v2.1 RC1 及之后版本）升级到 TiDB 3.0.x 或 master 版本。如果版本早于 TiDB 2.0.1 请选择如下两个方案：
 
-## 升级兼容性说明
+1. 停机升级，从当前版本直接升级到 TiDB 3.0.x。
+2. 先从当前版本滚动升级到 TiDB 2.0.1，再按照本文档滚动升级到 TiDB 3.0.x。
 
-- 不支持在升级后回退至 2.1.x 或更旧版本
-- 从 2.0.6 之前的版本升级到 3.0 之前，需要确认集群中是否存在正在运行中的 DDL 操作，特别是耗时的 `Add Index` 操作，等 DDL 操作完成后再执行升级操作
-- 2.1 及之后版本启用了并行 DDL，早于 2.0.1 版本的集群，无法滚动升级到 3.0，可以选择下面两种方案：
-    - 停机升级，直接从早于 2.0.1 的 TiDB 版本升级到 3.0
-    - 先滚动升级到 2.0.1 或者之后的 2.0.x 版本，再滚动升级到 3.0 版本
+## 注意事项
 
-> **注意：**
->
-> 在升级的过程中不要执行 DDL 请求，否则可能会出现行为未定义的问题。
+1. TiDB 3.0.x 版本和 master 版本兼容 [Kafka 版本的 TiDB Binlog](/v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md) 以及 [TiDB Binlog Cluster 版本](/v3.0/reference/tidb-binlog-overview.md)。
```

Comment:
```suggestion
1. TiDB 3.0.x 版本和 master 版本兼容 [TiDB Binlog Cluster 版本](/v3.0/reference/tidb-binlog-overview.md)。TiDB 3.0.1 版本兼容 [Kafka 版本的 TiDB Binlog](/v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md)，TiDB 3.0.2 及以上版本不再兼容 Kafka 版本的 TiDB Binlog。
```
This comment is from @liubo0127 : https://github.com/pingcap/docs-cn/pull/1875#discussion_r326996973